mesonbuild/munstable_coredata.py

```diff
@@ -91,17 +92,13 @@ def run(options):
             if v:
                 print('Native File: ' + ' '.join(v))
         elif k == 'compilers':
-            print('Cached native compilers:')
-            dump_compilers(v)
-        elif k == 'cross_compilers':
-            print('Cached cross compilers:')
-            dump_compilers(v)
+            for for_machine in MachineChoice:
+                print('Cached {} machine compilers:'.format(
+                    for_machine.get_lower_case_name()))
+                dump_compilers(v[for_machine])
         elif k == 'deps':
-            native = list(sorted(v.build.items()))
-            if v.host is not v.build:
-                cross = list(sorted(v.host.items()))
-            else:
-                cross = []
+            build = list(sorted(v.build.items()))
+            host = list(sorted(v.host.items()))
```

Comment:
__[Flake8]__ [F841] local variable 'host' is assigned to but never used ([link](https://sider.review/gh/repos/19784232/pulls/4010#issue-4097619))

<sub>posted by Sider</sub>